### PR TITLE
[BEAM-972] Add unit tests to Gearpump runner

### PR DESCRIPTION
**Pull request by huafengw**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by huafengw** [comment]
R: @kennknowles 
Hi, this is Huafeng from Gearpump team, I'm adding some unit tests to gearpump runner. Please help to review this pr.

**Comment by kennknowles** [comment]
Hi Huafeng! Thanks for contributing. I am on a little vacation but I I will take a look on Monday.

**Comment by huafengw** [comment]
@kennknowles Thanks, any comment?

File: runners/gearpump/src/test/java/org/apache/beam/runners/gearpump/PipelineOptionsTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    options.setEmbeddedCluster(cluster);
+    options.setParallelism(10);
+
+    byte[] serializedOptions = serialize(options);
+    GearpumpPipelineOptions deserializedOptions = new ObjectMapper()
+      .readValue(serializedOptions, PipelineOptions.class).as(GearpumpPipelineOptions.class);
+
+    assertNull(deserializedOptions.getEmbeddedCluster());
+    assertNull(deserializedOptions.getSerializers());
+    assertEquals(10, deserializedOptions.getParallelism());
```
Comment (kennknowles) [inline runners/gearpump/src/test/java/org/apache/beam/runners/gearpump/PipelineOptionsTest.java]: In Beam, our style is to use `assertThat(deserializedOptions.getParallelism(), equalTo(10))`.

**Comment by huafengw** [comment]
Thanks Kenn, I'll keep your advice in mind

**Pull request merged**